### PR TITLE
Limit and Skip issue note in Query reference document

### PR DESCRIPTION
**Pull request by varun-raj**
Signed-off-by: Varun Raj <varun@skcript.com>

This PR is to add a note for query reference documentation as the LIMIT and SKIP is not working in Fabric V1 & V1.1. Which helps developer know about this within the documentation. Refer [FAB-2809](https://jira.hyperledger.org/browse/FAB-2809)

## Checklist
 - [ ]  A link to the issue/user story that the pull request relates to
 - [ ]  How to recreate the problem without the fix
 - [ ]  Design of the fix
 - [ ]  How to prove that the fix works
 - [ ]  Automated tests that prove the fix keeps on working
 - [x]  Documentation - any JSDoc, website, or Stackoverflow answers?

## What documentation has been provided for this pull request
Website (Query Reference Page)

### CODE REVIEW

**Comment by varun-raj** [comment]
@EdProsser Can you please review this?

File: packages/composer-website/jekylldocs/reference/query-language.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ The `statement` property contains the defining rules of the query, and can have
 - `SKIP` is an optional operator which defines the number of results to skip.
 - `LIMIT` is an optional operator which defines the maximum number of results to return from a query, by default limit is set at 25.
 
+> Note: If you're using Fabric v1 or Fabric v1.1, the `LIMIT` and `SKIP` won't work as there is an issue passing the params to couchdb from fabric. refere : [FAB-2809](https://jira.hyperledger.org/browse/FAB-2809)
```
Comment (davidkel) [inline packages/composer-website/jekylldocs/reference/query-language.md]: As this doc is going into master, it should only refer to v1.1 of fabric. We don't refer to fabric directly in our docs but rather use a variable for substitution. 
{{site.data.conrefs.hlf_full}}
Also not sure about the "refere" word, did you mean "reference" ?

**Comment by varun-raj** [comment]
I've updated the content, can you please check it?

**Comment by davidkel** [comment]
@varun-raj Sorry, your DCO sign-off is missing.

**Comment by varun-raj** [comment]
@davidkel Can you check this now?

**Pull request merged**